polymath/extensions/pickler.py

```
@@ -90,920 +90,928 @@
+                # Unknown exception
+                else:
+                    raise
 
-            # Unknown exception
             else:
-                raise
-
+                # Raise any warnings
+                if _PICKLE_WARNINGS and first_exception is not None:
+                    if precision != initial_precision:
+                        warnings.warn('fpzip.compress increased precision from '
```

Ruff (0.14.7)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)